.github/workflows/build_push.yml

```diff
@@ -59,41 +57,139 @@ jobs:
           dependencies-cache-enabled: true
           configuration-cache-enabled: true
 
+      - name: Upload Jar
+        uses: actions/upload-artifact@v3
+        with:
+          name: jar
+          path: master/server/build/*.jar
+          if-no-files-found: error
+
+      - name: Upload icons
+        uses: actions/upload-artifact@v3
+        with:
+          name: icon
+          path: master/server/src/main/resources/icon
+          if-no-files-found: error
+
+      - name: Tar scripts dir to maintain file permissions
+        run: tar -cvzf scripts.tar.gz -C master/ scripts/
+      - name: Upload scripts.tar.gz
+        uses: actions/upload-artifact@v3
+        with:
+          name: scripts
+          path: scripts.tar.gz
+          if-no-files-found: error
+
+  bundle:
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [
+          linux-x64,
+          linux-all,
+          macOS-x64,
+          macOS-arm64,
+          # takes about 6m to build deb package
+          #debian-all,
```

Comment:
I think it would be fine if this was enabled for previews, it doesnt really mater how long a workflow takes

Reply:
do u mean enabled for releases? previews being fast is kinda useful sometimes

Reply:
Tachidesk isnt making builds like no tomorrow, so a little delay in previews isnt really important enough that we have to disable the creation of deb packages.

Reply:
You are right, someone can just comment out unwanted parts on their fork.